Lib/test/test_dbm_dumb.py

```diff
@@ -244,34 +244,27 @@ def test_eval(self):
     def test_warn_on_ignored_flags(self):
```

Comment:
Test name is no longer accurate, should be something like `test_raise_on_bad_flags`

Reply:
Good point!